cli/crates/cli/src/start/configuration/telemetry.rs

```diff
@@ -0,0 +1,9 @@
+#[derive(Debug, PartialEq, Clone, serde::Deserialize)]
+#[serde(deny_unknown_fields)]
+pub struct TelemetryConfig {
```

Comment:
I wonder do we even need to have this in the gateway level config? You can load the same file from your own config struct, and it will just skip all options it doesn't recognize.

Reply:
Hm I'd rather leave the file read part out of the tracing crate.